ports/ohmygodlookatthisknight/port.json

```
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "ohmygodlookatthisknight.zip",
+  "items": [
+    "OH MY GOD, look at this knight",
+    "ohmygodlookatthisknight"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "OH MY GOD, look at this knight",
+    "porter": [
+      "tabreturn"
+    ],
+    "desc": "Grab your shiny helmet and trusty sword, gather all the valor and honor you have, and show everyone what being the true knight actually means!",
+    "desc_md": null,
+    "inst": "Claim free on Steam (https://store.steampowered.com/app/781440/OH_MY_GOD_LOOK_AT_THIS_KNIGHT), download, and copy the game files to the ohmygodlookatthisknight/gamedata folder.",
+    "inst_md": "Claim free on [Steam](https://store.steampowered.com/app/781440/OH_MY_GOD_LOOK_AT_THIS_KNIGHT), download, and copy the game files to the `ohmygodlookatthisknight/gamedata` folder.",
+    "genres": [
+      "adventure"
+    ],
+    "image": null,
+    "rtr": false,
+    "exp": false,
+    "runtime": null,
+    "reqs": [],
+    "arch": [
+      "aarch64"
```

Looks like it is a 32bit gmloader port?

```suggestion
      "armhf"
```